src/js/components/lazy-load.js

```diff
@@ -107,20 +188,37 @@ class LazyLoad {
     return image
   }
 
-  destroy () {
-    $(window).off('scroll', this.onScrollHandler)
+  /**
+   * Parse string with breakpoint size and image sorce
```

Comment:
`Parse string with breakpoint size and image source in order to convert each breakpoint into an object.`